coral-component-masonry/examples/index.html

```diff
@@ -221,8 +221,25 @@ <h2 class="coral-Heading--M">Usage notes</h2>
         </div>
         <script>
           window.addEventListener('load', function() {
+            let uid = 0;
+            function getUID() {
+              return 'coral-masonry-' + uid++;
+            }
             const masonry = document.getElementById('masonry');
 
+            masonry.items.getAll().forEach(function (item) {
+              var article = item.querySelector('article');
+              article.setAttribute('role', 'presentation');
+              if (!item.id) {
+                item.id = getUID();
+              }
+              var contentId = article.id || item.id + '-content';
+              item.setAttribute('aria-labelledby', contentId);
```

Comment:
you can directly assign the value to article.id and use it to set the aria-labelledby.